### PR TITLE
Feature/service request urine lab

### PR DESCRIPTION
**Pull request by achen2401**
Address https://www.pivotaltracker.com/story/show/179694957
Changes include:

- Add a new section, titled "**Urine Drug Toxicology Screen**" under "Risk considerations" section, in the COSRI frontend app
- in CQL, include a new search for urine drug screen in service requests, in addition to the pre-existing search in observations.  The search will be matching on LOINC code if present, otherwise in the case of Skagit site, a custom system, i.e. **https://www.labcorp.com/tests**,  and code, i.e. **733727**, will be used for the search.
-  Add flag for when urine drug screen is absent and presence of at least one opioid medication

This is an example screenshot of the new section:
Screen Shot 2021-10-12 at 12 01 12 PM

### CODE REVIEW

File: src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql [hunk 1]
```diff
@@ -183,6 +185,10 @@ parameter InclusionMedicationsLookbackPeriod default 180 days //DO WE NEED THIS?
 
 //system url from PDMP identifier
 define PDMP_SYSTEM_VALUE: 'https://github.com/uwcirg/script-fhir-facade/'
+//system url for labcorp
+define LAB_CORP_SYSTEM_VALUE: 'https://www.labcorp.com/tests'
+//lab panel test order code from lab corp
+define Urine_Screen_Panel_Test_Order_Code: '733727'
```
Comment (pbugni) [inline src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql:191]: This is one of TWO panels they use.  Please add `763824`

File: src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql [hunk 2]
```diff
@@ -917,6 +924,37 @@ define ReportUrineDrugScreens:
...
+define ReportUrineLabServiceRequests:
+  ([ServiceRequest: "Urine drug screen for pain management"]) O
+  let c: O.code as FHIR.CodeableConcept
+  return {
+    Name: ConceptText(c),
+    Date: DateTimeText(O.authoredOn)
+  }
+  sort by Date desc
+
+// Urine drug screen from service request,  matching on custom system and concept code, e.g. those from Skagit site
```
Comment (pbugni) [inline src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql:937]: It's true this change is for Skagit, but it's potentially broader.

Probably more accurate to change these comments to "sites using labcorp for testing"

File: src/components/summary.json [hunk 3]
```diff
@@ -1094,6 +1043,64 @@
...
+                            "header": "Class",
+                            "targetValue":"opioid"
+                          }
+                        }
+                      ]
+                    }
+                  ]
+                },
+                "flagText": "There is no urine drug screen date yet record on this patient. Please check the first controlled substance prescription to determine if they are due for their 12 month drug screen."
+              }
```
[the remark this answers is not in the document]
Comment (achen2401) [reply src/components/summary.json:1100]: thank you, fixed

File: src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql (hunk 1 above)
Comment (achen2401) [reply src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql]: fixed.

File: src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql (hunk 2 above)
Comment (achen2401) [reply src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql]: thank you, fixed

File: src/icons/FlaskIcon.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+					<path d="M201.286,226.771c-3.657-3.657-9.143-3.657-12.8,0c-3.657,3.657-3.657,9.143,0,12.8l38.4,38.4
+						c1.829,1.829,3.657,2.743,6.4,2.743c1.829,0,4.571-0.914,6.4-2.743c3.657-3.657,3.657-9.143,0-12.8L201.286,226.771z"/>
+					<path d="M499.343,278.886l-19.2-19.2c-3.657-3.657-9.143-3.657-12.8,0l-19.2,19.2c-14.629,14.629-14.629,37.486,0,52.114
+						c7.314,7.314,16.457,10.971,25.6,10.971s18.286-2.743,25.6-10.971C513.971,316.371,513.971,293.514,499.343,278.886z
+						M486.543,318.2c-6.4,7.314-18.286,7.314-25.6,0c-7.314-7.314-7.314-18.286,0-25.6l12.8-12.8l12.8,12.8
+						C493.857,299.914,493.857,310.886,486.543,318.2z"/>
+				</g>
+			</g>
+		</g>
+	</svg>
```
Comment (ivan-c) [inline src/icons/FlaskIcon.js:41]: Could this be moved to an external `.svg` resource file?

**Review by ivan-c — APPROVED** [review]
Looks good! minor nitpick but ready to merge

**Pull request merged**